person.rb

```diff
@@ -35,6 +35,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
```

Comment:
- [ ] Again, `rental` should be first marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. 
- [ ] Next, create and initialize @rentals instance. This should be similar to the below example:
```ruby
class Person

  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
      @rentals = []
  end

  def add_rental(book, date)

    Rental.new(date, book, self)
  end

  .
  .
end
```